docs/plotting.md

```diff
@@ -2,100 +2,182 @@
+
 ``` bash
-python3 scripts/plot_dataframe.py -p BTC/ETH --indicators1 sma,ema --indicators2 macd
+freqtrade --strategy AwesomeStrategy plot-dataframe -p BTC/ETH --indicators1 sma ema --indicators2 macd
 ```
 
-### Advanced use
+### Further usage examples
 
-To plot multiple pairs, separate them with a comma:
+To plot multiple pairs, separate them with a space:
 
 ``` bash
-python3 scripts/plot_dataframe.py -p BTC/ETH,XRP/ETH
+freqtrade --strategy AwesomeStrategy plot-dataframe -p BTC/ETH XRP/ETH
 ```
 
-To plot a timerange (to zoom in):
+To plot a timerange (to zoom in)
 
 ``` bash
-python3 scripts/plot_dataframe.py -p BTC/ETH --timerange=20180801-20180805
+freqtrade --strategy AwesomeStrategy plot-dataframe -p BTC/ETH --timerange=20180801-20180805
 ```
 
-To plot trades stored in a database use `--db-url` argument:
+To plot trades stored in a database use `--db-url` in combination with `--trade-source DB`:
 
 ``` bash
-python3 scripts/plot_dataframe.py --db-url sqlite:///tradesv3.dry_run.sqlite -p BTC/ETH --trade-source DB
+freqtrade --strategy AwesomeStrategy plot-dataframe --db-url sqlite:///tradesv3.dry_run.sqlite -p BTC/ETH --trade-source DB
 ```
 
 To plot trades from a backtesting result, use `--export-filename <filename>`
 
 ``` bash
-python3 scripts/plot_dataframe.py --export-filename user_data/backtest_results/backtest-result.json -p BTC/ETH
-```
-
-To plot a custom strategy the strategy should have first be backtested.
-The results may then be plotted with the -s argument:
-
-``` bash
-python3 scripts/plot_dataframe.py -s Strategy_Name -p BTC/ETH --datadir user_data/data/<exchange_name>/
+freqtrade --strategy AwesomeStrategy plot-dataframe --export-filename user_data/backtest_results/backtest-result.json -p BTC/ETH
 ```
 
 ## Plot profit
 
-The profit plotter shows a picture with three plots:
+![plot-profit](assets/plot-profit.png)
+
+Plot profit shows an interactive graph with three plots:
 
 1) Average closing price for all pairs
 2) The summarized profit made by backtesting.
-   Note that this is not the real-world profit, but
-   more of an estimate.
-3) Each pair individually profit
+   Note that this is not the real-world profit, but more of an estimate.
+3) Profit for each individual pair
+
+The first graph is good to get a grip of how the overall market progresses.
 
-The first graph is good to get a grip of how the overall market
-progresses.
+The second graph will show if your algorithm works or doesn't.
+Perhaps you want an algorithm that steadily makes small profits, or one that acts less often, but makes big swings.
 
-The second graph will show how your algorithm works or doesn't.
-Perhaps you want an algorithm that steadily makes small profits,
-or one that acts less seldom, but makes big swings.
+The third graph can be useful to spot outliers, events in pairs that cause profit spikes.
 
-The third graph can be useful to spot outliers, events in pairs
-that makes profit spikes.
+Usage for the plot-profit module:
```

Comment:
"for the plot-profit subcommand"
or use "Possible arguments" here as above for plot-dataframe.

(I would also use the term 'option' instead of 'argument' everywhere in the text)

Reply:
Jup :+1: 

well, using option instead of argument might be confusing since `--help` uses "optional arguments:" as output.